src/core/runtime/config-browser.js

```diff
@@ -18,6 +18,10 @@ module.exports = () => ({
     }
   },
   Bootstrap: [
+    '/dnsaddr/bootstrap.libp2p.io/ipfs/QmNnooDu7bfjPFoTZYxMNLWUQJyrVwtbZg5gBMjTezGAJN',
+    '/dnsaddr/bootstrap.libp2p.io/ipfs/QmQCU2EcMqAqQPR2i9bChDtGNJchTbq5TbXJJ16u19uLTa',
+    '/dnsaddr/bootstrap.libp2p.io/ipfs/QmbLHAnMoJPWSCR5Zhtx6BHJX9KiKNN6tpvbUcqanj75Nb',
+    '/dnsaddr/bootstrap.libp2p.io/ipfs/QmcZf59bWwK5XFi76CZX8cbJ4BhTzzA3gU1ZjYZcYW3dwt',
```

Comment:
Are this working @mburns? I'm getting failed DNS lookups for bootstrap.libp2p.io.

```console
➜ nslookup
> set type=txt
> bootstrap.libp2p.io
Server:		1.1.1.1
Address:	1.1.1.1#53

Non-authoritative answer:
*** Can't find bootstrap.libp2p.io: No answer

Authoritative answers can be found from:
bootstrap.libp2p.io
	origin = ns1.dnsimple.com
	mail addr = admin.dnsimple.com
	serial = 1490150281
	refresh = 86400
	retry = 7200
	expire = 604800
	minimum = 300
> _dnslink.bootstrap.libp2p.io
Server:		1.1.1.1
Address:	1.1.1.1#53

** server can't find _dnslink.bootstrap.libp2p.io: NXDOMAIN
> dnslink.bootstrap.libp2p.io
Server:		1.1.1.1
Address:	1.1.1.1#53

** server can't find dnslink.bootstrap.libp2p.io: NXDOMAIN
```

Reply:
Can we connect to these with websockets (secure)? We cannot connect via tcp sockets from the browser.

Reply:
@jacobheun does libp2p support `dnsaddr` in node addresses currently?

Reply:
@alanshaw no not yet, resolve needs to be implemented in js-multiaddr in order to get the TXT records, https://github.com/multiformats/js-multiaddr/issues/94.

Reply:
Hi!

```
$ dig +short -t txt _dnsaddr.bootstrap.libp2p.io
"dnsaddr=/ip4/147.75.77.187/tcp/4001/ipfs/QmQCU2EcMqAqQPR2i9bChDtGNJchTbq5TbXJJ16u19uLTa"
"dnsaddr=/ip4/147.75.83.83/tcp/4001/ipfs/QmbLHAnMoJPWSCR5Zhtx6BHJX9KiKNN6tpvbUcqanj75Nb"
"dnsaddr=/ip4/147.75.94.115/tcp/4001/ipfs/QmcZf59bWwK5XFi76CZX8cbJ4BhTzzA3gU1ZjYZcYW3dwt"

"dnsaddr=/ip4/139.178.68.71/tcp/4001/ipfs/QmNnooDu7bfjPFoTZYxMNLWUQJyrVwtbZg5gBMjTezGAJN"
```

They're still being worked on / rolled out, but as of this writing, the first 3 results are working and my local go-ipfs instance can connect to them. The fourth will work Real Soon Now. Currently IPv4-only with IPv6 support also in-flight.

These nodes should be accessible over wss on `443` via nginx. Haven't tested that yet, [but it is all wired up](https://github.com/protocol/bifrost-infra/blob/master/ansible/roles/nginx_conf_bootstrappers/files/sites-enabled/1-libp2p-websocket.conf#L5-L8).